java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -555,12 +706,15 @@ private boolean handleMethodInvocation(Instruction instruction) {
             }));
         methodInvokedBehavior
           .exceptionalPathYields()
-          .forEach(yield ->
+          .forEach(yield -> {
+            org.sonar.plugins.java.api.semantic.Type exceptionType = yield.exceptionType();
             yield.statesAfterInvocation(
-              Lists.reverse(pop.values), Collections.emptyList(), pop.state, () -> constraintManager.createExceptionalSymbolicValue(yield.exceptionType())).forEach(ps -> {
-              ps.storeExitValue();
-              enqueue(new ProgramPoint(exitBlock), ps);
-            }));
+              Lists.reverse(pop.values), Collections.emptyList(), pop.state, () -> constraintManager.createExceptionalSymbolicValue(exceptionType)
+              ).forEach(ps -> {
+                ps.storeExitValue();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 15. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)